state/azure/tablestorage/tablestorage.go

```diff
@@ -53,44 +55,56 @@ import (
 const (
 	keyDelimiter        = "||"
 	valueEntityProperty = "Value"
-	operationTimeout    = 1000
 
-	accountNameKey = "accountName"
-	accountKeyKey  = "accountKey"
-	tableNameKey   = "tableName"
+	accountNameKey  = "accountName"
+	accountKeyKey   = "accountKey"
+	tableNameKey    = "tableName"
+	cosmosDBmodeKey = "cosmosDBmode"
```

Comment:
What's the correct capitalization for this? Should this be `cosmosDbMode`, `cosmosDBMode`...?